javascript/javascript1/week1/age-fy.js

```diff
@@ -0,0 +1,6 @@
+let yearOfBirth = 1993;
+let yearFuture = 2025;
+
+let age = yearFuture - yearOfBirth;
+
+console.log(" You will be " + age + " " + "years old in" + " " + yearFuture + ".");
```

Comment:
I guess if, you can arrange a call to help understand this because I don't see the back tick which is also used along with this concatenation. So would like to know this better. I wish this was explained in the class 1 with the example.